_posts/en/newsletters/2021-02-24-newsletter.md

```diff
@@ -0,0 +1,110 @@
+  activation parameters, most notably reducing the amount of hash rate
+  that needs to signal for the fork to activate from 95% to 90%.
+
+    Discussion continued about the LOT parameter on the mailing list,
+    mainly about whether or not it should be exposed to users as an
+    option they can set (or are forced to set).  No clear agreement was
+    reached as of this writing and there does not seem to be a widely
+    supported path forward to activating taproot---even though taproot
+    itself appears to be widely desired.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.12.1-beta.rc7][LND 0.12.1-beta] is a release candidate for a maintenance
+  release of LND.  It fixes an edge case that could lead to accidental
+  channel closure and a bug that could cause some payments to fail
+  unnecessarily, plus some other minor improvements and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19136][] wallet: add parent_desc to getaddressinfo FIXME:Xekyo
+
+- [Bitcoin Core #15946][] makes it possible to simultaneously use both
+  configuration options `prune` and `blockfilterindex` to maintain
+  [compact block filters][topic compact block filters] on a pruned node
+  (also serving them if the `peerblockfilters` configuration option is
+  used). An LND developer [indicated][osuntokun request] that this would
+  be beneficial for their software, and it could also allow a future
+  update that allows the wallet logic to use the block filters to
+  determine which historic blocks the pruned node needs to re-download
+  in order to import a wallet.
+
+- [Eclair #1693][] and [Rust-Lightning #797][] change how node address
+  announcements are handled.  The current [BOLT7][] specification
+  requires addresses be sorted within an announcement, however some
```

Comment:
to be or not to be?
```suggestion
  requires addresses to be sorted within an announcement, however some
```

Reply:
Both seem fine. I believe "The specification requires addresses be sorted" is an example of the mandative subjunctive.

See also: https://github.com/bitcoinops/bitcoinops.github.io/pull/290#discussion_r356024297 :)